### PR TITLE
Update image URLs for preview content.

### PR DESCRIPTION
**Pull request by jasmussen**
This PR updates the URLs for preview content to ones referencing s.w.org:

https://core.trac.wordpress.org/ticket/48292

Please test Image, Cover, Columns, Media & Text, and Gallery blocks in the Block Library, and verify they have image preview content.

### CODE REVIEW

**Review by gziolo — APPROVED** [review]
All urls load properly 👍

File: packages/block-library/src/gallery/index.js [hunk 1]
```diff
@@ -26,8 +26,8 @@ export const settings = {
 		attributes: {
 			columns: 2,
 			images: [
-				{ url: 'https://upload.wikimedia.org/wikipedia/commons/c/c3/Glacial_lakes%2C_Bhutan.jpg' },
-				{ url: 'https://upload.wikimedia.org/wikipedia/commons/0/01/Sediment_off_the_Yucatan_Peninsula.jpg' },
+				{ url: 'https://s.w.org/images/core/5.3/Glacial_lakes,_Bhutan.jpg' },
```
Comment (jorgefilipecosta) [inline packages/block-library/src/gallery/index.js]: Would it be safer to encode the comman in the URL as we did before?

Comment (jasmussen) [reply packages/block-library/src/gallery/index.js]: In addition to this, is it better to do `//s.w.org/images` than `https://s.w.org/images`?

I don't know the answer to either my question, or the comma one, but let me know what to encode the commas to and I can do it.

Comment (jasmussen) [reply packages/block-library/src/gallery/index.js]: Oh, I guess it's `%2C`, right?

Comment (jorgefilipecosta) [reply packages/block-library/src/gallery/index.js]: Yes, the URL with comma encoded is https://s.w.org/images/core/5.3/Glacial_lakes%2C_Bhutan.jpg


I am also not aware if we should encode it or not but I normally see commas encoded so maybe there is some reason to encode them.
I guess we can merge with them encoded and then ask on the ticket what URL should be used and if needed we can update.

> In addition to this, is it better to do //s.w.org/images than https://s.w.org/images?

Good question I am also not sure what is the common practice I guess we can https for now.

**Pull request merged**